platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/AbstractGossip.java

```diff
@@ -154,13 +156,14 @@ protected AbstractGossip(
             @NonNull final EventObserverDispatcher eventObserverDispatcher,
             @NonNull final StatusActionSubmitter statusActionSubmitter,
             @NonNull final Consumer<SignedState> loadReconnectState,
-            @NonNull final Runnable clearAllPipelinesForReconnect) {
-
+            @NonNull final Runnable clearAllPipelinesForReconnect,
+            @NonNull final Time time) {
```

Comment:
For the sake of consistency, I think `time` would be better placed as the 3rd parameter, after `threadManager`. That's where it is for the gossip subclasses